.eslintrc.cjs

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
I got a warning about this file not being included in `tsconfig.json`, so I fixed that. That resulted in prettier and eslint running over this, which standardized the quotes and formatting.